packages/wonder-blocks-clickable/components/clickable.md

```diff
@@ -0,0 +1,86 @@
+### Creating a Clickable component
+
+You can make custom components Clickable by simply returning them in function of the Clickable child. The eventState parameter the function takes allows access to states pressed, hovered and clicked, which you may use to create custom styles.
+
+```jsx
+const {StyleSheet} = require("aphrodite");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {MemoryRouter, Route, Switch} = require("react-router-dom");
+const Color = require("@khanacademy/wonder-blocks-color").default;
+const {Body} = require("@khanacademy/wonder-blocks-typography");
+
+const styles = StyleSheet.create({
+    focused: {
+        border: "none",
+    },
+    hovered: {
+        textDecoration: "underline",
+    },
+    pressed: {
+        color: Color.teal,
+    },
+});
+
+<View>
+    <Clickable onClick={() => alert("You clicked some text!")}>
+        {
+            eventState =>
+                <Body style={[
```

Comment:
Can you wrap this in a View to make it look more like a card and change the background color of it as the eventState changes?